my-app/src/services/employees.service.js

```diff
@@ -0,0 +1,62 @@
+import Constants from "../constants/constants.service";
+import authService from "./auth.service";
+import departmentsService from "./departments.service";
+
+const baseUrl = Constants.getBaseUrl();
+
+const getEmployees = () => {
+  const options = {
+    method: "GET",
+    headers: {
+      authorization: authService.getToken(),
+    },
+  };
+
+  return fetch(`${baseUrl}/employees`, options).then(async (res) => {
+    if (!res.ok) {
+      throw new Error(res.statusText);
+    }
+
+    const data = await res.json();
```

Comment:
If you use await, why dont you use the async construction?

Reply:
.then(async (res) => {
    if (!res.ok) {
      throw new Error(res.statusText);
    }

    const data = await res.json();


It's there=)